src/test/java/org/apache/commons/collections4/bloomfilter/BitCountProducerFromDefaultIndexProducerTest.java

```diff
@@ -41,24 +37,19 @@ protected int getBehaviour() {
         return AS_ARRAY_DISTINCT | AS_ARRAY_ORDERED;
     }
 
-    @Test
-    @Disabled("Current behaviour will return the same index twice, each with a count of 1")
-    public final void testFromIndexProducer() {
-
-        BitCountProducer producer = createProducer();
-        Map<Integer, Integer> m = new HashMap<>();
+    @Override
+    protected int[][] getExpectedBitCount() {
+        return new int[][]{{0, 1}, {63, 1}, {1, 1}, {1, 1}, {64, 1}, {127, 1}, {128, 1}};
+    }
 
-        producer.forEachCount((i, v) -> {
-            m.put(i, v);
-            return true;
-        });
+    @Override
+    protected int[] getExpectedIndex() {
+        return expected;
```

Comment:
If the Abstract test is modified as suggested to separately test behaviour and the expected indices then this can `return data`.